docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-wide-guide-card/index.scss

```diff
@@ -53,9 +53,13 @@
 	}
 	.nci-guide-card__description {
 		@include u-margin-bottom(3);
-		/* START Remove the typeset override when ticket #4355 is completed */
-		@include typeset("body", "xs");
-		/* END Code Removal Note */
+		@include typeset($theme-prose-font-family);
+		line-height: line-height($theme-body-font-family, $theme-body-line-height);
```

Comment:
The line-height should go into the typeset, which should also contain the font-size.